src/components/ebay-button/test/test.server.js

```diff
@@ -26,6 +26,18 @@ Object.keys(properties).forEach(property => {
     });
 });
 
+test('renders button with default type', context => {
+    const input = { };
+    const $ = testUtils.getCheerio(context.render(input));
+    expect($(`button.btn`).attr('type')).to.equal('button');
+});
+
+test('renders button with overriden type', context => {
```

Comment:
Can there be an additional test for `reset` or does that seem redundant since the test already checks for `submit`? 
Following up on that, since there are only 3 valid types `button`, `submit` and `reset` should there be a check for this in the renderer?

Reply:
Good point, but I think we're treating this as roughly an HTML attribute pass-through, and we don't normally validate those. I.e. you can do `aria-checked="maybe"` and we won't stop you.